symphony/bdk/core/service/datafeed/on_disk_datafeed_id_repository.py

```diff
@@ -0,0 +1,78 @@
+from abc import ABC, abstractmethod
+from pathlib import Path
+from symphony.bdk.core.config.model.bdk_config import BdkConfig
+
+
+class DatafeedIdRepository(ABC):
```

Comment:
This file is called `on_disk_datafeed_id_repository.py`, however you define the `DatafeedIdRepository` interface + the `OnDiskDatafeedIdRepository` implem. 

I'd move those classes to `symphony/bdk/core/service/datafeed/__init__.py` to make imports simpler

Reply:
@symphony-thibault I understand you don't want to put the interface and implementation in the same module. But I don't like putting code in the __init__.py files.
We could split this into two modules and put import statements in the __init__.py files to simplify imports.